templates/email/default/waste/echo_failed_bulky_payment.txt

```diff
@@ -0,0 +1,40 @@
+Subject: Echo failed to confirm payment
+
+[%
+USE pounds = format('%.2f');
+PROCESS 'waste/_bulky_data.html';
+~%]
+
+Echo failed to confirm payment for the following bulky waste collection:
```

Comment:
The language is the wrong way round - after successful payment, the user is redirected to us, we note this on the collection report as an update, and we then send that update to Echo to confirm payment has been received. It is if that update to Echo fails that we need to send this email. So this and other bits like that will need rewording. "We failed to update Echo about confirmed payment" etc